shuffle/cli/src/shared.rs

```diff
@@ -407,44 +413,61 @@ pub struct NetworksConfig {
 }
 
 impl NetworksConfig {
-    pub fn default() -> Self {
+    pub fn default() -> Result<Self> {
         let mut network_map = BTreeMap::new();
-        network_map.insert("localhost".to_string(), Network::default());
-        NetworksConfig {
+        network_map.insert("localhost".to_string(), Network::localhost()?);
+        network_map.insert("trove".to_string(), Network::trove_testnet()?);
```

Comment:
let's have the key match the call. There might be a trove production that we will want to call trove.

```rust
        network_map.insert("trove_testnet".to_string(), Network::trove_testnet()?);
```